modules/claims_api/app/services/claims_api/bgs_to_lighthouse_claims_mapper_service.rb

```diff
@@ -2,71 +2,46 @@
 
 module ClaimsApi
   class BGSToLighthouseClaimsMapperService < ClaimsApi::Service
-    attr_accessor :bgs_claims, :lighthouse_claims
+    attr_accessor :bgs_claim, :lighthouse_claim
 
-    def initialize(bgs_claims:, lighthouse_claims:)
-      @bgs_claims        = bgs_claims
-      @lighthouse_claims = lighthouse_claims
+    def initialize(bgs_claim: nil, lighthouse_claim: nil)
+      @bgs_claim        = bgs_claim
+      @lighthouse_claim = lighthouse_claim
     end
 
     def process
-      return [] unless claims_exist?
+      return matched_claim if bgs_and_lighthouse_claims_exist?
+      return unmatched_bgs_claim if bgs_claim_only?
+      return unmatched_lighthouse_claim if lighthouse_claim_only?
```

Comment:
nitpick: `unmatched_bgs_claim` and `unmatched_lighthouse_claim` probably don't need `unmatched_` prepended, since the `matched_claim` and `bgs/lighthouse_claim_only` names make it clear